src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -36,12 +36,31 @@
 
     private static final long serialVersionUID = 1;
 
-    public PullRequestSCMHead(int number) {
+    private String user;
```

Comment:
> No, that is something else: the person the PR is “assigned to”, which in most PRs is not set at all.

I meant that `assigne` does represent a `user`.

When you use the [API](https://developer.github.com/v3/users/#get-a-single-user) for `user` entity: `/users/:username`

You have to do the query using `username`, what it means the `login`.

Anyway, this is not a bug at all. @imeredith you decide.